charts/add-ons/grafana/templates/grafana.yaml

```diff
@@ -128,7 +128,7 @@ spec:
         # see https://github.com/grafana/grafana/issues/20096
         - name: GODEBUG
           value: netdns=go
-        image: {{.Values.image.name}}:{{default .Values.global.linkerdVersion .Values.global.controllerImageVersion}}
+        image: {{.Values.image.name}}:{{ default (default .Values.global.linkerdVersion .Values.global.controllerImageVersion) .Values.image.tag}}
```

Comment:
> .Values.image.tag

Should this be `.Values.grafana.image.tag`?

Reply:
@ihcsim As its the sub-chart, we don't actually need `.grafana` for referencing values as helm makes sure we do that, and for CLI we manually do the same.